app/components/history_component.html.erb

```diff
@@ -0,0 +1,73 @@
+<ol class=" relative border-s border-slate-200 dark:border-slate-700 ">
+  <% @data.each_with_index do |changes, index| %>
+    <li class="mb-10 ms-4">
+      <div
+        class="
+          absolute
+          w-3
+          h-3
+          bg-slate-200
+          rounded-full
+          mt-1.5
+          -start-1.5
+          border
+          border-white
+          dark:border-slate-700
+          dark:bg-slate-700
+        "
+      ></div>
+      <time
+        class="
+          mb-1
+          text-sm
+          font-normal
+          leading-none
+          text-slate-500
+          dark:text-slate-400
+        "
+      ><%= changes[:updated_at] %></time>
+      <h2 class=" text-lg font-semibold text-slate-900 dark:text-white ">
+        <% unless @url.nil? %>
+          <%= link_to t(:"components.history.link_text", version: changes[:version]),
+          path_with_params(@url, { version: changes[:version] }),
+          data: {
+            turbo_stream: true
+          },
+          class: "text-slate-800 dark:text-white hover:underline" %>
+          <% if index == 0 %>
+            <span
+              class="
+                bg-green-200
+                text-green-800
+                text-xs
+                font-medium
+                ml-2
+                px-2.5
+                py-0.5
+                rounded-full
+                dark:bg-green-900
+                dark:text-green-300
+              "
+            ><%= t(:"components.history.latest") %></span>
```

Comment:
We have a viral component for this. `<%= viral_pill(text: t(:"components.history.latest"), color: :green, classes: "ml-2") %>` can replace this block

Reply:
Thanks I didn't even realize we had that! Updated in 37ef7d21eeef1cc770b73ca98b30f2a942fc7b9e